tests/test_args.py

```diff
@@ -103,8 +104,11 @@ def test_config_file_parsing(
     handler = ArchConfigHandler()
     arch_config = handler.arch_config
 
+    # TODO: Use the real values from the test fixture instead of clearing out the entries
+    arch_config.disk_config.device_modifications = []
+
     assert arch_config == ArchConfig(
-        version='3.0.0',
+        version=archinstall.__version__,
```

Comment:
This will break the tests every time there's a new version released

Reply:
Right, I'll need to have a deeper look into it. Lets get it in for now as the PR is a good improvement, thanks for that